versions/management/commands/import_reviews.py

```diff
@@ -0,0 +1,241 @@
+from bs4 import BeautifulSoup
+import djclick as click
+import requests
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import Q
+
+from libraries.utils import generate_fake_email
+from versions.models import Review, ReviewResult
+
+User = get_user_model()
+
+
+@click.command()
+@click.option(
+    "--dry-run", is_flag=True, help="Parse the data but don't save to database"
+)
+@click.option(
+    "--dry-run-users", is_flag=True, help="Save reviews, but don't link users"
+)
+def command(dry_run, dry_run_users):
+    """Import Boost library reviews from boost.org table data"""
+
+    url = "https://www.boost.org/community/review_schedule.html"
+    response = requests.get(url)
+    soup = BeautifulSoup(response.text, "html.parser")
+
+    # Parse both tables
+    scheduled_review_table = soup.find("table", summary="Formal Review Schedule")
+    past_review_table = soup.find("table", summary="Review Results")
+
+    upcoming_reviews = _parse_table(scheduled_review_table)
+    past_reviews = _parse_table(past_review_table, past_results=True)
+
+    click.secho(
+        f"Found {len(upcoming_reviews)} upcoming and {len(past_reviews)} past reviews"
+    )
+
+    if dry_run:
+        click.secho("Dry run - no changes made")
+        return
+
+    # Import everything in a transaction
+    with transaction.atomic():
+        # Create or update past reviews
+        for review_data, results in past_reviews:
```

Comment:
Might it be worth adding some logging here?

Reply:
To be clear, not just there, maybe added throughout?

Reply:
Sure. Added a handful.